src/workerd/api/node/zlib-util.h

```diff
@@ -234,11 +245,44 @@ class ZlibUtil final: public jsg::Object {
     }
   };
 
-  uint32_t crc32Sync(kj::Array<kj::byte> data, uint32_t value);
+  struct Options {
+    jsg::Optional<int> flush;
+    jsg::Optional<int> finishFlush;
+    jsg::Optional<int> chunkSize;
+    jsg::Optional<kj::uint> windowBits;
+    jsg::Optional<int> level;
+    jsg::Optional<kj::uint> memLevel;
+    jsg::Optional<kj::uint> strategy;
+    jsg::Optional<kj::Array<kj::byte>> dictionary;
+    // We'll handle info on the JS side for now
+    jsg::Optional<kj::uint> maxOutputLength;
+
+    JSG_STRUCT(flush,
+        finishFlush,
+        chunkSize,
+        windowBits,
+        level,
+        memLevel,
+        strategy,
+        dictionary,
+        maxOutputLength);
+    JSG_MEMORY_INFO(Options) {}
```

Comment:
```suggestion
```

Reply:
What does that do btw? It came in when I copied and pasted some stuff

Reply:
it has to do with collecting memory details for a heap snapshot. Most of the time you won't need it.